core/packages/test/scripts/deploy-ethereum.sh

```diff
@@ -38,18 +38,15 @@ start_lodestar() {
             -H 'Content-Type: application/json' \
             -d '{"jsonrpc": "2.0", "id": "1", "method": "eth_getBlockByNumber","params": ["0x0", false]}' | jq -r '.result.hash')
 
-        if [[ "$OSTYPE" =~ ^darwin ]]
-        then
-            timestamp=$(gdate -d'+10second' +%s)
-        else
-            timestamp=$(date -d'+10second' +%s)
-        fi
+        timestamp=$(date -d'+10second' +%s)
 
         npx lodestar dev \
             --genesisValidators 8 \
             --genesisTime $timestamp \
             --startValidators "0..7" \
-            --enr.ip "127.0.0.1" \
+            --enr.ip6 "::1" \
```

Comment:
What's the reasoning behind making both geth and lodestar bind to ipv6 interfaces? Usually ipv4 is the best default, since its more widely supported. For example, not sure if this will work in Docker or Github Actions runners, etc

I vaguely recall geth preferentially binding to ipv6 interfaces due to config in `/etc/hosts`. If that's the case though, maybe we need to sort that out instead.

Reply:
ipv6 was a shortcut to finding more differences between system configs, though maybe commenting out `/etc/hosts` would be simpler. Happy to switch to ipv4, just want to pick one to reduce confusion.

Reply:
@doubledup so what is the error thrown when not explicted add `127.0.0.1` in the setup?  I met the [issue](https://github.com/Snowfork/snowbridge/pull/771#issuecomment-1417907080) in one of my physical machine 
and resolved after Clara updates `lodestar` to `1.4.2`, not sure if will also resolve the issue here.

Reply:
Hardhat and lodestar had trouble talking to geth, I can't remember the exact error.

I believe `localhost` gets mapped to an address in `/etc/hosts`, but we don't want to assume anything about system-level config like `/etc/hosts`. eg. using ipv6 & localhost had issues because on our Linux boxes localhost is only mapped to `127.0.0.1`, while the Macs also map it to `::1`.